e2e/src/app.e2e-spec.ts

```diff
@@ -71,6 +73,8 @@ describe('Test App', () => {
             const readEle = await page.getReadValueFieldFromValueComponent(valueEleComp);
             expect(await readEle.getText()).toEqual('3');
 
+            // browser.sleep(200000);
```

Comment:
Should this be left in? I think it's used to keep the browser open when running the tests locally, right?

Reply:
yes, I find it helpful as a comment so next time I do not have to look for the command